aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,190 @@
+
+## Benefits of using .NET native AOT with ASP.NET Core
+
+Using the .NET native AOT deployment model provides the following benefits:
+
+* **Minimize disk footprint**; when publishing using native AOT a single executable is produced containing just the code from external dependencies that is used to support the program. Reduced executable size can lead to smaller container images (in containerized deployment scenarios) which can reduce deployment time.
+* **Reduced startup time**; native AOT applications can show reduced start-up times which means the application is ready to service requests quicker. This can also help during deployment where container orchestrators need manage transition from one version of the application to another.
+* **Reduce memory demand**; native AOT applications can have reduced memory demands depending on the nature of the work being performed by the application. This reduced memory consumption can lead to greater deployment density and improved scalability.
+
+## When using .NET native AOT with ASP.NET Core should be avoided
+
+Not all features in ASP.NET Core are currently compatible with .NET native AOT. The following table summarizes ASP.NET Core feature compatibility with .NET native AOT:
+
+| Feature                       | Fully Supported | Partially Supported | Not Supported        |
+| ----------------------------- | --------------- | ------------------- | -------------------- |
+| Minimal APIs                  |                 | Yes                 |                      |
+| MVC                           |                 |                     | No                   |
+| Blazor                        |                 |                     | No                   |
+| SignalR                       |                 |                     | No                   |
+| Authentication                |                 |                     | No (JWT coming soon) |
+
+It is important to test application functionality thoroughly when moving to a native AOT deployment model to ensure that functionality observed during developmnet (using the runtime) is preserved in the native executable.
+
+In addition to API compatability, deploying ASP.NET Core apps (without native AOT) can be more efficient in some scenarios. For example, when a container image is created for an app that is based on the ASP.NET Core base images, an additional layer is created containing just the binaries which are often smaller than the native AOT binary. The nodes that host the container images might contain a cached copy of the base layers of the container image. If they do, the time to download the additional layer can be small.The benefits of .NET native AOT deployment of ASP.NET Core apps depends heavily on the details of the deployment strategy.
+
+
+## Known issues
+
+### 8.0-preview3
```

Comment:
I think it's better to link to the issue in the repo here instead of having to keep the two places in sync.